x-pack/plugins/synthetics/public/apps/synthetics/components/monitors_page/overview/overview_page.tsx

```diff
@@ -52,14 +51,6 @@ export const OverviewPage: React.FC = () => {
     }
   }, [dispatch, locationsLoaded, locationsLoading]);
 
-  // fetch overview for query state changes
-  useEffect(() => {
-    if (pageState.query !== query) {
-      dispatch(fetchMonitorOverviewAction.get({ ...pageState, query }));
-      dispatch(setOverviewPageStateAction({ query }));
-    }
-  }, [dispatch, pageState, query]);
-
```

Comment:
Removed this as there's another effect below it which serves the same purpose (except watching the `query`). The `query` change is being watched by `useMonitorFiltersState` hook,  which is called inside `useMonitorList` hook.  `useMonitorList` is called below on this page.

`useMonitorFiltersState` syncs the query and filter states among Overview, Management and Url.